waltid-services/waltid-wallet-api/src/main/resources/example_klogging.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
this looks different from klogging.conf inside the test folder
is it on purpose?